utils/multer.js

```diff
@@ -0,0 +1,44 @@
+const multer = require('multer')
+const multerConstant = require('../constants/multer')
+const multerMemoryStorage = multer.memoryStorage()
+
+const MB_1 = multerConstant.FILE_SIZE_1MB
+const profileFileSize = multerConstant.PROFILE_FILE_SIZE
+
+function fileFilterImagesOnly (req, file, cb) {
+  const mimetype = file.mimetype
+  const isMimeTypePng = (mimetype === 'image/png')
+  const isMimeTypeJpeg = (mimetype === 'image/jpeg')
+  if (isMimeTypePng || isMimeTypeJpeg) {
+    return cb(null, true)
+  }
```

Comment:
I know this will allow images with `jpeg` extension, but will it allow `jpg`? 🤔 
Also, can we not store the allowed file types in an array maybe and then check if the `mimetype` is any of the array elements? (using `.includes` method on array) 
Would reduce the hardcoding, right?

Reply:
In image/jpeg subtype jpg is included